extension/app/src/hooks/useFileUploaderService.ts

```diff
@@ -0,0 +1,395 @@
+import type {
+  ConversationPublicType,
+  FileUploadedRequestResponseType,
+} from "@dust-tt/client";
+import { useSendNotification } from "@dust-tt/sparkle";
+import type {
+  LightWorkspaceType,
+  Result,
+  SupportedFileContentType,
+} from "@dust-tt/types";
+import {
+  Err,
+  isSupportedFileContentType,
+  isSupportedImageContentType,
+  MAX_FILE_SIZES,
+  Ok,
```

Comment:
Can you use (& add if necessary) from dust-client ?
(ok and err are there, supported content types too)